inc/ui/namespace.php

```diff
@@ -31,16 +32,26 @@ function bootstrap() {
  * to the admin bar.
  */
 function register_menu() {
-	$hook = add_submenu_page(
+	// Add top level page.
+	$hook = add_menu_page(
 		null,
+		__( 'Documentation', 'altis' ),
+		'edit_posts',
+		PAGE_SLUG
+	);
+
+	// Add our default dev docs.
+	$dev_set = get_documentation_set( 'dev-docs' );
```

Comment:
@roborourke 
OK. I did miss this bit. But it raises a couple of conflicts between the issue description and the acceptance criteria.

The original issue says:

> This would allow building other types of documentation using the infrastructure we already have, but without adding new groups and pages to the built-in UI. i.e. I could add a project set, add some groups for different types of user docs, and then choose to render that on some other page myself, without it rendering on the default admin page.

But the acceptance criteria says:

> New top level admin menu section for "Documentation", each set is added [as] a sub page

and 

> "Documentation" top level page in the admin only renders docs from first available set

So on one hand, we should add each set as a sub page, but the description says the additional docs set page location could be somewhere else.

On the other hand, if the dev docs set is only rendered on local/dev environments, on other environments we could end up with an empty top level documentation menu, even if we do have other sets. Unless we do add the additional sets to the default menu. 

I think a solution could be:

 - [ ] If no sets are found (non local/dev environment and no additional sets), don't render documentation menu
 - [ ] If additional sets are added, add the Documentation Menu and add them as sub pages.
 - [ ] Always render the first set found: dev-docs on local/dev environment, first additional set if any added
 - [] If someone wants to add their documentation set to another menu, the can do so by not adding to the default array of sets. Instead, adding their own menu/submenu and still use the provided functionality/filters to have the system render their docs. (I need to test this will work) 

How does that sound?

Reply:
Well caught re. the original description. I had missed that and had a different idea of what would be good for this feature, apologies for that.

Regarding your proposed solution:

- > If no sets are found (non local/dev environment and no additional sets), don't render documentation menu
   
   Perfect, makes sense
- > If additional sets are added, add the Documentation Menu and add them as sub pages.
   
   Again, yep this makes sense to me.
- > Always render the first set found: dev-docs on local/dev environment, first additional set if any added
   
   Yep agreed. I think we can leave the ordering up to individual preference and always just use whatever set is first in the array as the default docs page
- > If someone wants to add their documentation set to another menu, the can do so by not adding to the default array of sets. Instead, adding their own menu/submenu and still use the provided functionality/filters to have the system render their docs. (I need to test this will work)
   
   Yeah that could work too, seems like it'll provide ultimate flexibility 🤔  Depends how tied into being part of the admin the UI is to work really. If the output doesn't have to be on an admin page to render and function then your idea here is the best one. The only alternative I could suggest is having a `show_in_menu` property on the `Set` or an optional `admin_menu_callback` that could replace the default `add_submenu_page()` logic.

Great suggestions, and thanks for bearing with. I'll be more thorough with the acceptance criteria in future and note any deviations from the original idea.